src/server/scripts/Spells/spell_quest.cpp

```diff
@@ -2482,6 +2482,62 @@ class spell_q13665_q13790_bested_trigger : public SpellScriptLoader
         }
 };
 
+// herald of war
+class spell_59064_portal_orgrimmar : public SpellScriptLoader
+{
+public:
+    spell_59064_portal_orgrimmar() : SpellScriptLoader("spell_59064_portal_orgrimmar") { }
+
+    class spell_59064_portal_orgrimmar_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_59064_portal_orgrimmar_SpellScript);
+
+        void HandleScript(SpellEffIndex /*effIndex*/)
+        {
+            if (Unit* target = GetHitUnit())
```

Comment:
So i can do this then?
[code]
GetHitUnit()->CastSpell(GetHitUnit(), uint32(GetEffectValue()));
[/code]